spec/features/admin/merchants/index_spec.rb

```diff
@@ -16,19 +16,27 @@
     expect(page).to have_content(@merchant3.name)
   end
 
-  #   As an admin,
-# When I click on the name of a merchant from the admin merchants index page (/admin/merchants),
-# Then I am taken to that merchant's admin show page (/admin/merchants/:merchant_id)
-# And I see the name of that merchant
   # US 25
   it "clicking the name redirects to merchant admin show page and displays the name" do
     visit admin_merchants_path
 
-    expect(page).to have_link("#{@merchant1.name} ##{@merchant1.id}")
-    click_link("#{@merchant1.name} ##{@merchant1.id}")
-
+    expect(page).to have_link("#{@merchant1.name}, merchant ##{@merchant1.id}")
+    click_link("#{@merchant1.name}, merchant ##{@merchant1.id}")
     expect(current_path).to eq(admin_merchant_path(@merchant1.id))
-
     expect(page).to have_content(@merchant1.name)
   end
-end
\ No newline at end of file
+
+  #US 27/28
```

Comment:
This test satisfies US 27, but not US 28. I'd add another test with a sections of disabled & enabled merchants that sort by the boolean. We'd expect to see a status display of "Enable" or "Disabled"